democracy/views/utils.py

```diff
@@ -199,3 +203,112 @@ def to_internal_value(self, data):
             else:
                 raise ValidationError(_('Image size should be smaller than {} bytes.'.format(settings.MAX_IMAGE_SIZE)))
         raise ValidationError(_('Invalid content. Expected "data:image"'))
+
+
+class TranslatableSerializer(serializers.Serializer):
+    """
+    A serializer for translated fields.
+
+    translated_fields must be declared in the Meta class.
+    By default, translation languages obtained from settings, but can be overriden
+    by defining translation_lang in the Meta class.
+    """
+
+    def __init__(self, *args, **kwargs):
+        assert hasattr(self.Meta, 'translated_fields'), 'Improperly configured: Need to specify translated_fields'
```

Comment:
Why is there a need for listing the field separately in `fields` and `translated_fields` in Meta? Surely, the model contains information on which fields are translated, so we can add to `translated_fields` all those fields that are listed in `fields` and are contained in translations?